src/charm.py

```diff
@@ -157,6 +167,28 @@ def _csr_exists_in_requirer(self, csr: str, relation_id: int) -> bool:
                 return True
         return False
 
+    def _relation_id_parameter_valid(
+        self, found_relation_ids: List[int], requested_relation_id: str
+    ) -> str:
+        """Validates the given CSR with the requirers.
```

Comment:
The docstring doesn't make much sense, we're not checking the CSR here

Reply:
It's difficult to write a one liner for this. Updated it.